bedrock/base/templates/includes/protocol/navigation/menu-firefox/join.html

```diff
@@ -14,7 +14,7 @@
           <li>
             <section class="mzp-c-menu-item mzp-has-icon">
               <a class="mzp-c-menu-item-link" href="{{ url('firefox.accounts') }}" data-link-name="Join Firefox" data-link-type="nav" data-link-position="topnav" data-link-group="join">
-                <img class="mzp-c-menu-item-icon" src="{{ static('protocol/img/logos/firefox/logo.svg') }}" class="mzp-c-menu-item-icon" alt="" width="24" height="24">
+                <img class="mzp-c-menu-item-icon" src="{{ static('node_modules/@mozilla-protocol/core/protocol/img/logos/firefox/logo.svg') }}" class="mzp-c-menu-item-icon" alt="" width="24" height="24">
```

Comment:
@Errolyn I haven't pulled this PR locally, but one thing to consider here is I don't believe we ship the content of `node_modules` in the production image for bedrock. So even though this may work locally, once deployed it may have issues still? Static assets like this should probably be kept in `/media/`, since that root directory is served via the CDN and cached more heavily if the filename is hashed.

It may be worth considering keeping the current behavior of copying the protocol assets over to /media. You may be able to achieve this using something like the [copy-webpack-plugin](https://www.npmjs.com/package/copy-webpack-plugin) perhaps?